.github/workflows/golang-build.yaml

```diff
@@ -36,6 +36,17 @@ jobs:
         version: latest
         args: -v --timeout 5m
         skip-cache: true
+  test:
+      name: Unit test
+      runs-on: ubuntu-latest
+      steps:
+        - name: Checkout code
+          uses: actions/checkout@v4
+        - name: Install Go
+          uses: actions/setup-go@v4
+          with:
+            go-version: '1.20'
+        - run: make test
```

Comment:
Just a note is that the the `build.Tests` step also runs the unit tests, although these are not as visible as your job.

Reply:
So can I assume that Build.Test doesn't really need it? I deleted it.